<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskChoiseDialog</class>
 <widget class="QDialog" name="taskChoiseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#taskChoiseDialog
{
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="30,20,10,120,20,20,20" columnstretch="1,5,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>67</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,200">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label_titleLeftBotton_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>SourceHanSansK-Regular</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: bold large 'SourceHanSansK-Regular';font-size:20px;
}</string>
       </property>
       <property name="text">
        <string>参数设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>2048</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QGridLayout" name="gridLayout" rowstretch="55,50,55,50,55,50,55" columnstretch="74,230,35,74,230">
     <property name="horizontalSpacing">
      <number>18</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 75 14pt ;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>参数一</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_function1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: rgba(0, 0, 0, 0);
border-radius: 6px; 
border: 2px solid rgb(1,159,164);
font: 75 18pt ;
color: rgb(0, 0, 0);
padding: 5 20px;

}

QLineEdit:enabled:hover, QLineEdit:enabled:focus {
/* 选中高亮背色 */
border: 2px solid rgb(245,166,35);
}


</string>
       </property>
       <property name="text">
        <string>50.0</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 75 14pt ;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>参数二</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="edit_function2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: rgba(0, 0, 0, 0);
border-radius: 6px; 
border: 2px solid rgb(1,159,164);
font: 75 18pt ;
color: rgb(0, 0, 0);
padding: 5 20px;

}

QLineEdit:enabled:hover, QLineEdit:enabled:focus {
/* 选中高亮背色 */
border: 2px solid rgb(245,166,35);
}


</string>
       </property>
       <property name="text">
        <string>1.00</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 75 14pt ;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>参数三</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edit_function3">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: rgba(0, 0, 0, 0);
border-radius: 6px; 
border: 2px solid rgb(1,159,164);
font: 75 18pt ;
color: rgb(0, 0, 0);
padding: 5 20px;

}

QLineEdit:enabled:hover, QLineEdit:enabled:focus {
/* 选中高亮背色 */
border: 2px solid rgb(245,166,35);
}


</string>
       </property>
       <property name="text">
        <string>60.000</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 75 14pt ;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>参数四</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="edit_function4">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: rgba(0, 0, 0, 0);
border-radius: 6px; 
border: 2px solid rgb(1,159,164);
font: 75 18pt ;
color: rgb(0, 0, 0);
padding: 5 20px;

}

QLineEdit:enabled:hover, QLineEdit:enabled:focus {
/* 选中高亮背色 */
border: 2px solid rgb(245,166,35);
}


</string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 75 14pt ;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>参数五</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edit_function5">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: rgba(0, 0, 0, 0);
border-radius: 6px; 
border: 2px solid rgb(1,159,164);
font: 75 18pt ;
color: rgb(0, 0, 0);
padding: 5 20px;

}

QLineEdit:enabled:hover, QLineEdit:enabled:focus {
/* 选中高亮背色 */
border: 2px solid rgb(245,166,35);
}


</string>
       </property>
       <property name="text">
        <string>500.000</string>
       </property>
       <property name="maxLength">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 75 14pt ;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>参数六</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="edit_function6">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
background-color: rgba(0, 0, 0, 0);
border-radius: 6px; 
border: 2px solid rgb(1,159,164);
font: 75 18pt ;
color: rgb(0, 0, 0);
padding: 5 20px;

}

QLineEdit:enabled:hover, QLineEdit:enabled:focus {
/* 选中高亮背色 */
border: 2px solid rgb(245,166,35);
}


</string>
       </property>
       <property name="text">
        <string>2.000</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 75 14pt ;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>参数七</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="com_function7">
       <property name="styleSheet">
        <string notr="true">QComboBox{font: large rgb(255, 255, 255);font-size:14px;border-width:2px;
	color: rgb(0,0,0);
	background-color: rgba(255, 255, 255, 0);
	border-color:rgba(1,159,164,1);
	border-style: solid;
	border-radius:6px;
	 padding: 5px 20px;
	font: 75 18pt ;
	color: rgb(0, 0, 0);
}

QComboBox::down-arrow {image: url(:/images/Down_normalBlue.png);}
QComboBox::down-arrow:hover{image: url(:/images/Down_hover.png);}
QComboBox::down-arrow:pressed{image: url(:/images/Down_normalBlue.png);}

QComboBox::drop-down {
subcontrol-origin: padding;
subcontrol-position: top right;
width: 30px;
border-left-width: 0px;
border-left-color: gray;
border-left-style: solid;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;}</string>
       </property>
       <item>
        <property name="text">
         <string>预设参数一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>预设参数二</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>预设参数三</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);
font: 75 14pt ;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>参数八</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QComboBox" name="com_function8">
       <property name="styleSheet">
        <string notr="true">QComboBox{font: large rgb(255, 255, 255);font-size:14px;border-width:2px;
	color: rgb(0,0,0);
	background-color: rgba(255, 255, 255, 0);
	border-color:rgba(1,159,164,1);
	border-style: solid;
	border-radius:6px;
	 padding: 5px 20px;
	font: 75 18pt ;
	color: rgb(0, 0, 0);
}

QComboBox::down-arrow {image: url(:/images/Down_normalBlue.png);}
QComboBox::down-arrow:hover{image: url(:/images/Down_hover.png);}
QComboBox::down-arrow:pressed{image: url(:/images/Down_normalBlue.png);}

QComboBox::drop-down {
subcontrol-origin: padding;
subcontrol-position: top right;
width: 30px;
border-left-width: 0px;
border-left-color: gray;
border-left-style: solid;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;}</string>
       </property>
       <item>
        <property name="text">
         <string>预设参数一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>预设参数二</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="80,100,80,100,80">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clearDate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ font: bold large 'SourceHanSansK-Regular';font-size:14px;
                                border: 0px solid;
                                background-color:rgba(255, 30, 30, 1);
                                color:rgba(255, 255, 255,1);
                                border-radius: 10px;
                                }
                                QPushButton:disabled{
                                background-color:#DCDCDC;
                                border-radius:10px;
                                color:#FFFFFF;    
                                }
                                QPushButton:hover{
                                border: 0px solid;
                                background-color:rgba(255, 87, 87,1);
                                color:rgba(255, 255, 255,1);
                                border-radius:10px;
                                opacity:0.2;
                                }
                                QPushButton:pressed{
                                background-color:rgba(56, 60, 73,1);
                                color:rgba(255, 255, 255,1);
                                border-radius:10px;
                                }</string>
       </property>
       <property name="text">
        <string>返回上级</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_export">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{ font: bold large 'SourceHanSansK-Regular';font-size:14px;
                                border: 0px solid;
                                background-color:rgb(1, 160, 165);
                                color:rgba(255, 255, 255,1);
                                border-radius: 10px;
                                }
                                QPushButton:disabled{
                                background-color:#DCDCDC;
                                border-radius:10px;
                                color:#FFFFFF;    
                                }
                                QPushButton:hover{
                                border: 0px solid;
                                background-color:rgba(88, 164, 176,1);
                                color:rgba(255, 255, 255,1);
                                border-radius:10px;
                                opacity:0.2;
                                }
                                QPushButton:pressed{
                                background-color:rgba(58, 108, 116,1);
                                color:rgba(255, 255, 255,1);
                                border-radius:10px;
                                }</string>
       </property>
       <property name="text">
        <string>参数确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
